deploy_pages.sh

```diff
@@ -16,7 +16,7 @@ git clone "$REPO" "$TEMP_DIR"
 cd "$TEMP_DIR"
 
 # Remove old contents, copy new contents into place, commit and push everything
-git checkout -b gh-pages
+git checkout gh-pages
```

Comment:
I've looked at what others are doing. To be extra sure, they use `git checkout -t origin/gh-pages`.

Reply:
I've tested both variants and both seem to work correctly.